src/main/java/com/conveyal/analysis/controllers/GtfsTileController.java

```diff
@@ -56,103 +52,112 @@
  *
  * TODO handle cancellation of HTTP requests (Mapbox client cancels requests when zooming/panning)
  */
-public class GtfsTileController implements HttpController {
+public class GtfsTileController {
```

Comment:
Yea, originally this was implemented as it's own `HttpController` but I modified it to be used by the `GtfsController` since there is some commonality. I assumed there might be common vector tile methods that would be abstracted out of this file and the `NetworkTileController` eventually and would take another pass at cleaning them up and organizing them.